R/retry.R

```diff
@@ -91,6 +91,10 @@ backoff_full_jitter <- function(i, resp, pause_base = 1, pause_cap = 60,
       error_description <- ""
       status <- status_code(resp)
     }
+    if (status == 429) {
+      length_429 <- as.numeric(resp$headers[["retry-after"]])
+      if (!is.null(length_429) && length_429 >= pause_min) length <- length_429
```

Comment:
I don't think the logic is quite right here - `length_429` will never be `NULL` because you coerce it to numeric

Inside the `if` statement, I'd prefer something like `length <- max(pause_min, length_429)`